notebooks/community/model_garden/model_garden_video_object_tracking_bytetrack.ipynb

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
<p>Rewrite:</p><p>video object tracking model on Google Cloud's Vertex AI by integrating an image object detection (IOD) endpoint with the open-source <a href="https://github.com/ifzhang/ByteTrack" target="_blank">ByteTrack</a> object tracking algorithm.  -&gt;</p><p> </p><p>video object tracking model on Vertex AI Endpoint resource with the open-source <a href="https://github.com/ifzhang/ByteTrack" target="_blank">ByteTrack</a> object tracking algorithm. </p><p> </p><p>Set up a Vertex AI IOD endpoint&nbsp; -&gt; Set up a Vertex AI Endpoint resource, with</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2098/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>